### PR TITLE
using original ContextHash, too

### PR DESCRIPTION
**Pull request by tinect**
the original cache context hash is currently missing.

### CODE REVIEW

File: Subscriber/CacheKeyExtender.php [hunk 1]
```diff
@@ -47,23 +47,23 @@ public static function getSubscribedEvents()
...
         }
 
         $session = $this->dependencyProvider->getSession();
         $userId = (int)$session->get('sUserId', 0);
-        $result = $this->getCustomerPriceGroupId($userId);
+        $priceGroup = $this->getCustomerPriceGroupId($userId);
 
-        if ($result !== 0) {
-            $priceGroup = $result;
+        if ($priceGroup == 0) {
```
Comment (mitelg) [inline Subscriber/CacheKeyExtender.php]: please use type safe comparison `===`
to ensure the type, you could cast the customer price group id

**Review by mitelg — CHANGES_REQUESTED** [review]

File: Subscriber/CacheKeyExtender.php (hunk 1 above)
Comment (tinect) [reply Subscriber/CacheKeyExtender.php]: Hey @mitelg 
sure :-)

**Comment by tinect** [comment]
ping :-)

**Review by mitelg — APPROVED** [review]

**Comment by mitelg** [comment]
thanks for your PR @tinect :+1: :tada: :blue_heart:

**Pull request closed without merging**